### PR TITLE
Add Airbrake reporting for before and after Angular loads

### PR DESCRIPTION
**Pull request by cvializ**
@andrewconner @ajkochanowicz 

I decided against using the ngHelperAirbrake because it was just a very thin wrapper that didn't provide enough functionality to justify the extra dependency.

It debounces for 5 seconds, but that's open to change.

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/dev.html [hunk 1]
```diff
@@ -19,6 +19,23 @@
   mixpanel.init("abb7e1226370392c849ec16fadff2584");*/
 </script>
 <script src="/dist/lib.js"></script>
+<!--
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/dev.html:22]: Great. Make sure to add this to `index.html` as well.

Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/dev.html:22]: I'm blind.

**Comment by andrewconner** [comment]
Looks good. I'm sure we're about to be surprised with what comes out.

File: kifi-backend/modules/shoebox/angular/dev.html (hunk 1 above)
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/dev.html:22]: ; )

**Pull request merged**